datafusion/common/src/column.rs

```diff
@@ -71,7 +71,11 @@ impl Column {
         }
     }
 
-    fn from_idents(idents: &mut Vec<String>) -> Option<Self> {
+    /// Create a Column from multiple normalized identifiers
+    ///
+    /// For example, `foo.bar` would be represented as a two element vector
+    /// `["foo", "bar"]`
+    fn from_idents(mut idents: Vec<String>) -> Option<Self> {
```

Comment:
Changed the signature to be easier to work with (pass the vec by ownership rather than &mut)